gdpr_api/tests/test_gdpr.py

```diff
@@ -0,0 +1,106 @@
+import uuid
+from unittest import mock
+
+import pytest
+import requests_mock
+from django.conf import settings
+from django.urls import reverse
+
+from gdpr_api.tests.conftest import get_api_token_for_user_with_scopes
+from pysakoinnin_sahk_asiointi.models import User
+
+
+def _gdpr_action(method, scope, user, query_uuid=None):
+    if not query_uuid:
+        query_uuid = user.uuid
+    with requests_mock.Mocker() as req_mock:
+        gdpr_profile_url = reverse("api-1.0.0:get_user_info", kwargs={"user_id": query_uuid})
+        auth_header = get_api_token_for_user_with_scopes(
+            user.uuid, [scope], req_mock
+        )
+        return method(gdpr_profile_url, headers={"Authorization": auth_header})
+
+
+def _gdpr_query(api_client, user, query_uuid=None):
+    return _gdpr_action(api_client.get, settings.GDPR_API_QUERY_SCOPE, user, query_uuid)
+
+
+def _gdpr_delete(api_client, user, query_uuid=None):
+    return _gdpr_action(api_client.delete, settings.GDPR_API_DELETE_SCOPE, user, query_uuid)
+
+
+@pytest.fixture
+def user():
+    return User(uuid=uuid.uuid1())
```

Comment:
I think it would be better to use uuid v4 unless there is some actual need for v1.

Reply:
```suggestion
    return User(uuid=uuid.uuid4())
```

I'm fine either way, but I'd like to see the justification for the suggestion. :grin: 

The story behind uuid1 is that I copied it from https://github.com/City-of-Helsinki/django-helusers/blob/19ffdc6fbf4f8046ffb0473c095aeacd393464a7/helusers/models.py#L61 :thinking:

Reply:
Well, because of [theoretic privacy concerns](https://github.com/tc39/proposal-uuid/tree/master/analysis#background) regarding uuid v1. In this case they are of course very theoretical, but I still think a good rule of thumb is to not use v1 unless that is really needed. And actually in that case nowadays it is probably better to use v7 instead 🙂